### DISCUSSION TITLE
A way to group processes of the same name, similar to Task Manager

### DISCUSSION THREAD

**Discussion by Cornelius-Figgle** [nicolargo/glances#2178] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/110970234/199179090-3a5e9d57-c489-4ae9-ae96-8f663cf6b3eb.png)
![image](https://user-images.githubusercontent.com/110970234/199179127-fb55b82f-4823-4f4b-ba08-cac7ce777b35.png)

Would be nice if Glances could group tasks together if they have the same name, so we can see total resources for the whole of the program

**Comment by nicolargo** [comment, marked as answer]
Yes it is already possible to join the threads into program by clicking on **j** (for the moment only in console mode (not available in the WebUI), or using the --program option from the command line .

I let the issue open to trace the implementation the 'j' hotkey on the WebUI

**Reply by Cornelius-Figgle** [reply]
> I let the issue open

ok cool, didn't realise it already existed
